sklearn/datasets/tests/test_covtype.py

```diff
@@ -23,3 +23,22 @@ def test_fetch(fetch_covtype_fxt):
     # test return_X_y option
     fetch_func = partial(fetch_covtype_fxt)
     check_return_X_y(data1, fetch_func)
+
+
+def test_fetch_asframe(fetch_covtype_fxt):
+    pd = pytest.importorskip('pandas')
+    bunch = fetch_covtype_fxt(as_frame=True)
+    frame = bunch.frame
```

Comment:
The assert below will be useful because the test will already crash here if frame does not exist

Reply:
DONE.